service-worker.js

```diff
@@ -97,11 +101,17 @@ function fetchEventListener(event) {
                 // The namespace defines the type of content. See http://www.openzim.org/wiki/ZIM_file_format#Namespaces
                 // TODO : read the contentType from the ZIM file instead of hard-coding it here
                 if (nameSpace === 'A') {
-                    // It's an article
-                    contentType = 'text/html';
+                    if (regexpVTT.test(title)) {
+                        // It's a subtitle
+                        contentType = 'text/vtt';
+                    }
```

Comment:
@mossroy Apologies, ignore previous message, I see you repeat the test, once in A namespace and once in - namespace.

Reply:
Yes.
I'm wondering if I should test the file extensions first, and ignore the namespace to set the content-type. I think I still need to fallback to text/html in the A namespace (because not 100% sure every HTML page inside this namespace ends with .html or .htm)

Reply:
I think it's safest to test the namespace, even if it means repeating the test. So this is fine.